test/e2e/reconciler/cluster/syncerheartbeat_test.go

```diff
@@ -1,49 +0,0 @@
-/*
-Copyright 2022 The KCP Authors.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-
-package cluster
-
-import (
-	"context"
-	"testing"
-
-	workloadv1alpha1 "github.com/kcp-dev/kcp/pkg/apis/workload/v1alpha1"
-	"github.com/kcp-dev/kcp/test/e2e/framework"
-)
-
-func TestSyncerHeartbeat(t *testing.T) {
```

Comment:
why is this gone? Was useless?

Reply:
See https://github.com/kcp-dev/kcp/pull/909/files#diff-91621de609392e23db5fc8ea76095bd9e1fd7cfefc8d496188205888f316210eR55

Reply:
Yeah, it's effectively subsumed by the new syncer test. Also, the syncer fixture is checking for a ready cluster which is implicitly validating the heartbeat behavior.